packages/app-audit-logs/src/views/Logs/Table/styled.tsx

```diff
@@ -10,7 +10,12 @@ export const ActionWrapper = styled("div")<{ value: string }>`
     border-radius: 5px;
 
     ${({ value, theme }) => {
-        if (value === "CREATE" || value === "PUBLISH" || value === "SUCCESS") {
+        if (
+            value === "CREATE" ||
+            value === "PUBLISH" ||
+            value === "SUCCESS" ||
+            value === "RESTORE_FROM_TRASH"
+        ) {
```

Comment:
1) Use the `ActionType` enum from `config.ts`
2) Create an array of allowed action types, and use `actionTypes.includes(value)` instead of the `||` operator.